Library/Strings.swift

```diff
@@ -727,6 +727,28 @@ contributeurs"
       substitutions: ["backers_count": Format.wholeNumber(backers_count)]
     )
   }
+  /**
+   "%{backers_count}
+backers"
+
+   - **en**: "%{backers_count}
+backers"
+   - **de**: "%{backers_count}
+Unterstützer"
+   - **es**: "%{backers_count}
+patrocinadores"
+   - **fr**: "%{backers_count}
+contributeurs"
+   - **ja**: "%{backers_count} バッカー"
+  */
+  public static func Backers_count_separator_backers_other(backers_count: String) -> String {
```

Comment:
Looks like the size of this PR is mostly caused by the generated strings...was it intentional to include these as well?
Let's say we now use one of these we won't even show their default (English) representation but the key instead.

Should we pair on this?